### PR TITLE
Update a11y.js for 2025

### PR DESCRIPTION
**Pull request by mgifford**
Resolves \#1

This will expand it to include Drupal 7 and a little CMS called WordPress that might have a bit of traction.

---
**Test websites**:

- https://wordpress.org ?
- https://backdropcms.org ?

### CODE REVIEW

File: dist/a11y.js [hunk 1]
```diff
@@ -215,7 +215,7 @@ return JSON.stringify({
     };
   }),
   screen_reader_classes: captureAndLogError(() => {
-    return document.querySelectorAll('.sr-only, .visually-hidden').length > 0;
+    return document.querySelectorAll('.sr-only, .visually-hidden', '.screen-reader-text', 'element-invisible').length > 0;
```
Comment (joedolson) [inline dist/a11y.js:218]: I believe this should be `.element-invisible`, per https://www.drupal.org/node/2022859

Comment (mgifford) [reply dist/a11y.js:218]: Oops.. Thanks @joedoleson..

Comment (mgifford) [reply dist/a11y.js:218]: Oops.. Thanks @joedoleson..

Comment (tunetheweb) [reply dist/a11y.js:218]: So can we update this?

```suggestion
    return document.querySelectorAll('.sr-only, .visually-hidden', '.screen-reader-text', '.element-invisible').length > 0;
```

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**